graphprot/DataSet.py

```diff
@@ -8,7 +8,7 @@
 import h5py
 import copy
 
-from .community_pooling import community_detection, community_pooling
+from community_pooling import community_detection, community_pooling
```

Comment:
you do need the dot no ?

Reply:
well it probably works without but I would leave it so that we make sure that we refer to the file here